### DISCUSSION TITLE
`f:fast-usdc` a3p tests are failing but CI still passes

### DISCUSSION THREAD

**Discussion by anilhelvaci** [Agoric/agoric-sdk#10654] [Q&A] [answered]
## Context
The reason why I raise this as a "Discussion" question is that I'm not sure if this is the intended behavior and worth opening an issue for it. So please let me know if this is an issue, I'll convert to it immediately. 

## Problem Definition
a3p tests for `f:fast-usdc` currently check that `agoricNames.instance` has a property called `fastUsdc`. In the current version of master (`a66f6a1408016746f4f480241f3d6efa53732841`) `start-fast-usdc.js` core eval fails so `fastUsdc` does not end up in `agoricNames.instance`.

I have noticed that this behavior goes unnoticed when `yarn test -m fast-usdc` is run. I also noticed that in the CI as well. Here's a recent CI run that passed however `f:fast-usdc/test.sh` fails. 

https://github.com/Agoric/agoric-sdk/actions/runs/12225954791/job/34100571062?pr=10643#step:10:179

Same thing in https://github.com/Agoric/agoric-sdk/pull/10609 => https://github.com/Agoric/agoric-sdk/actions/runs/12225957386/job/34100576932?pr=10609#step:10:178

When `set -ueo pipefail` added on top of `test.sh`, `yarn test -m fast-usdc` fails.

## Next Steps
Please let me know if this is intended or deserves an issue. @dckc @turadg @Chris-Hibbert

**Comment by dckc** [comment]
a fix is underway in

 - https://github.com/Agoric/agoric-sdk/pull/10641

I'd like to think a new issue isn't worthwhile unless I neglect to land that in the next day or so.

**Comment by dckc** [comment, marked as answer]
oh. meanwhile, #10641 isn't linked to an issue, so maybe it would help to convert this one right away

**Reply by anilhelvaci** [reply]
Done. See https://github.com/Agoric/agoric-sdk/issues/10657

**Reply by anilhelvaci** [reply]
Also update #10641 description as well
